components/raftstore/src/store/fsm/peer.rs

```diff
@@ -1114,6 +1114,7 @@ where
                     "peer_id" => self.fsm.peer_id(),
                     "res" => ?res,
                 );
+                res.metrics.written_query_num += 1;
```

Comment:
only the enum in the ExecResult (such as CompactLog, ChangePeer, IngestSST, DeleteRange...)  will have the apply result. such as the `Put` `DELETE` command does not have it. you can check the code:
https://github.com/tikv/tikv/blob/06c3e76eab84482d49a36d634fb2a6c05995f54b/components/raftstore/src/store/fsm/apply.rs#L1390-L1409